sql/_23_apricot_qa/_03_i18n/tr_TR/_09_identifiers/cases/_007_identifiers_index_cardinality.sql

```diff
@@ -23,9 +23,9 @@ INSERT INTO Çç_Ğğ_İı_Öö_Şş_Üü VALUES (2,3,3,'+Çç_Ğğ_İı','+Çç
 show index in Çç_Ğğ_İı_Öö_Şş_Üü;
 
 --test
-SELECT INDEX_CARDINALITY('Çç_Ğğ_İı_Öö_Şş_Üü','i_Çç_Ğğ_İı_Öö_Şş_Üü_Çç_Öö',0);
+SELECT INDEX_CARDINALITY('dba.Çç_Ğğ_İı_Öö_Şş_Üü','i_Çç_Ğğ_İı_Öö_Şş_Üü_Çç_Öö',0);
 --test
-SELECT INDEX_CARDINALITY('Çç_Ğğ_İı_Öö_Şş_Üü','i_Çç_Ğğ_İı_Öö_Şş_Üü_Çç_Öö',1);
+SELECT INDEX_CARDINALITY('dba.Çç_Ğğ_İı_Öö_Şş_Üü','i_Çç_Ğğ_İı_Öö_Şş_Üü_Çç_Öö',1);
 --test
 SELECT INDEX_CARDINALITY('Çç_Ğğ_İı_Öö_Şş_Üü','i_Çç_Ğğ_İı_Öö_Şş_Üü_Çç_Öö',2);
```

Comment:
Below, the dba user must be specified.
```
--test
SELECT INDEX_CARDINALITY('dba.Çç_Ğğ_İı_Öö_Şş_Üü','i_Çç_Ğğ_İı_Öö_Şş_Üü_Çç_Öö',2);
--test
SELECT INDEX_CARDINALITY('dba.Çç_Ğğ_İı_Öö_Şş_Üü23','i_Çç_Ğğ_İı_Öö_Şş_Üü_Çç_Öö',1);
```

Reply:
Reflected the review. Please check again.